components/02-molecules/cards/custom-card/_yds-custom-card.scss

```diff
@@ -0,0 +1,107 @@
+@use '../../../01-atoms/atoms';
+@use '../../../00-tokens/tokens';
+
+.custom-card {
+  @include atoms.clickable-component-heading-link;
+
+  --card-border: 1px solid var(--color-gray-200);
+
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  column-gap: var(--size-spacing-6);
+
+  [data-collection-featured='false'] &,
+  &[data-with-image='false'] {
+    grid-template-columns: 1fr;
+  }
+
+  @media (min-width: tokens.$break-s) {
+    &[data-with-image='true'] {
+      [data-collection-featured='true'] & {
+        grid-template-columns: repeat(2, 1fr);
+      }
+
+      [data-collection-featured='false'] & {
+        grid-template-columns: 1fr 2fr;
+      }
+    }
+  }
+  @media (min-width: tokens.$break-m) {
+    &:hover {
+      box-shadow: var(--drop-shadow-level-1);
+    }
+
+    flex-basis: 100%;
+    display: flex;
+    flex-direction: column;
```

Comment:
Switch these two chunks. Even though there's no conflicts here, if you set an attribute in both places, the hover one would never work because of source order, so states like hover, focus, etc. and modifiers like breakpoints etc. should always come after the "base" styles.